trans: [-0.269103595227, -0.512540873107, -0.0121790399571, ]
rot: [0.00484068334654, 0.00441495878449, 0.703015991972, 0.711143860942, ]
rot_euler: [0.0130927936255, -0.00052681396194, 1.55929802549, ]
parent: /ar_marker_6
child: /panda_link0
